.github/workflows/main.yml

```diff
@@ -27,86 +18,44 @@ jobs:
       - name: "Install PHP"
         uses: "shivammathur/setup-php@2.4.3"
         with:
-          coverage: "none"
-          php-version: "${{ matrix.php-version }}"
-
-      - name: "Validate composer.json and composer.lock"
-        run: "composer validate --strict"
+          coverage: none
+          php-version: "7.4"
+          tools: composer:v2
```

Comment:
The composer v2 dependency makes use of parallel downloads of dependencies, therefore makes the general setup process faster.

See also https://github.com/laravel/framework/pull/33158#issue-431613254